.gitignore

```diff
@@ -97,3 +97,4 @@ uiDesigner.xml
 lets-plot-images
 documentation_resources/raw_tutorials/.ipynb_checkpoints
 src/test/kotlin/biokotlin/kmer/KmerDemo.kt
+b73.gff
```

Comment:
What are the files in the dotoutput folder?  How are these different from files in testData?  If this the output of running tests?  Should these be in gitignore?  They are further down in the pull request, and changes to them are indicated, but not shown.  I'm wondering if these below in the repository.